.github/workflows/test-fuzz.yaml

```diff
@@ -11,8 +11,8 @@ jobs:
     env:
       GOOS: ${{ matrix.goos }}
       GOARCH: ${{ matrix.goarch }}
-      GOFLAGS: '-trimpath -mod=readonly'
-      GO111MODULE: 'on'
+      GOFLAGS: "-trimpath" # -mod=readonly
```

Comment:
@tie We tried to fix the problem differently but after spending some time on it, we didn't find something conclusive. Do you think you could have a look? 

Side note: it appears that the bug forcing us to use go 1.14.9 for go fuzz has been fixed. I did try to update it but ran into another issue. For the context, the generated tests present in this PR need go embed and thus Go 1.16. To move forward, we committed those for now.